idea/common/src/interfaces/api-requests.ts

```diff
@@ -17,9 +19,16 @@ interface GetOutgoingMessagesParams extends IGenesis, IPaginationParams, Pick<IM
 interface GetAllProgramsParams extends IGenesis, IPaginationParams, SearchParam {
   publicKeyRaw?: string;
   owner?: string;
+  fromDate?: string;
+  toDate?: string;
```

Comment:
I guess it's better to move it to a separate enum and extend the rest from it

Reply:
Yeah, I think so